color_helper.sublime-settings

```diff
@@ -338,6 +338,18 @@
                 "meta.tag.any.html meta.attribute-with-value.style.html"
             ]
         },
+        {
+            "name": "JavaScript/JSX",
+            "base_scopes": [
+                "source.js",
+                "source.jsx",
+            ],
```

Comment:
Presumably this could also cover TypeScript, by adding the following:

```suggestion
            "name": "JavaScript/JSX/TypeScript",
            "base_scopes": [
                "source.js",
                "source.jsx",
                "source.ts",
            ],
```

I'm not a TypeScript developer however, and don't really have the time to give this a whirl at the moment.